src/convertToObject.js

```diff
@@ -12,7 +12,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleCss = {};
+
+  const filteredString = sourceString
+    .split(';')
+    .map(style => style.split(':'))
+    .filter(style => style.length === 2);
+
+  for (const style of filteredString) {
```

Comment:
[CODE PRACTICE] - Avoid using loops in this task, use iteration methods instead.